tests/integration-tests/tests/storage/test_fsx_lustre.py

```diff
@@ -38,21 +38,26 @@ def test_fsx_lustre(region, pcluster_config_reader, clusters_factory, s3_bucket_
     cluster = clusters_factory(cluster_config)
     remote_command_executor = RemoteCommandExecutor(cluster)
 
-    _test_fsx_lustre_correctly_mounted(remote_command_executor, mount_dir)
+    _test_fsx_lustre_correctly_mounted(remote_command_executor, mount_dir, os)
     _test_import_path(remote_command_executor, mount_dir)
     _test_fsx_lustre_correctly_shared(remote_command_executor, mount_dir)
     _test_export_path(remote_command_executor, mount_dir, bucket_name)
 
 
-def _test_fsx_lustre_correctly_mounted(remote_command_executor, mount_dir):
+def _test_fsx_lustre_correctly_mounted(remote_command_executor, mount_dir, os):
     logging.info("Testing fsx lustre is correctly mounted")
     result = remote_command_executor.run_remote_command("df -h -t lustre | tail -n +2 | awk '{print $1, $2, $6}'")
     assert_that(result.stdout).matches(r"[0-9\.]+@tcp:/fsx\s+3\.4T\s+{mount_dir}".format(mount_dir=mount_dir))
 
     result = remote_command_executor.run_remote_command("cat /etc/fstab")
+    if os == "centos7":
```

Comment:
can we move this mapping into a dict so that it's easy to extend support for other OSs? Also mount_options might not be initialized if we run this on a non supported os. We probably want to handle that case by defaulting to "" and asserting that mount_options is not empty (just so we have a clear error message in case tests are ran against non supported os)

Reply:
done and done